Feature/37 terraform created files in archive (#38)
terraform-google-modules/terraform-google-event-function: CHANGELOG.md

## [Unreleased]

### Added

> - New for `source_dependent_files` var. If used `archive` won't be created until terraform created `local_file`s are finished. [#38]

Reviewer: ```suggestion
- The `source_dependent_files` variable. If used `archive` won't be created until Terraform created `local_file`s are finished. [#38]
```
